### PR TITLE
pacman-mirrors: add SJTUG mirror

### PR DESCRIPTION
**Pull request by skyzh**

### CODE REVIEW

**Comment by lazka** [comment]
Did you build the package?

**Comment by skyzh** [comment]
> Did you build the package? You need to update the checksums.

Thanks for reminding me, I'll look into instructions...

**Comment by skyzh** [comment]
@lazka I've updated PKGBUILD. PTAL, thanks!

**Comment by lazka** [comment]
sorry, I forgot, can you also update the pkgver to the current date?

@elieux OK with adding this mirror?

**Comment by skyzh** [comment]
> sorry, I forgot, can you also update the pkgver to the current date?
> 
> @elieux OK with adding this mirror?

Done.

By the way, here's our mirror contact information.

Location: Shanghai, China (we mainly serve users in southeast China)
Operator: Shanghai Jiao Tong University Linux User Group (SJTUG)
Contact: sjtug-mirror-maintainers@googlegroups.com
URL: https://mirrors.sjtug.sjtu.edu.cn/msys2/ (https only)

**Comment by lazka** [comment]
> @elieux OK with adding this mirror?

I assume it's OK

File: pacman-mirrors/mirrorlist.msys [hunk 1]
```diff
@@ -12,3 +12,4 @@ Server = https://mirrors.tuna.tsinghua.edu.cn/msys2/msys/$arch/
 Server = http://mirrors.ustc.edu.cn/msys2/msys/$arch/
 Server = http://mirror.bit.edu.cn/msys2/msys/$arch/
 Server = https://mirror.selfnet.de/msys2/msys/$arch/
+Server = https://mirrors.sjtug.sjtu.edu.cn/msys2/$arch/
```
Comment (skyzh) [inline pacman-mirrors/mirrorlist.msys:15]: Sorry for the mistake, just now I found that this URL should be /msys2/msys/$arch. I'll make a PR in a day or two.

Comment (lazka) [reply pacman-mirrors/mirrorlist.msys:15]: no problem, thanks!

**Pull request merged**